modules/terraform-cdp-aws-pre-reqs/main.tf

```diff
@@ -205,12 +209,12 @@ resource "aws_vpc_endpoint" "interface_endpoints" {
     if var.create_vpc && var.create_vpc_endpoints
   }
 
-  vpc_id              = local.vpc_id
+  vpc_id              = module.aws_cdp_vpc.vpc_id
   service_name        = data.aws_vpc_endpoint_service.interface_endpoints[each.key].service_name
   vpc_endpoint_type   = "Interface"
```

Comment:
@jimright vpc_endpoint_type = "Interface" and private_dns_enabled = true static parameters. Could there be use-cases for other configurations? These parameters are relevant only when create\_vpc = true, is that correct?

Reply:
> vpc_endpoint_type = "Interface" and private_dns_enabled = true static parameters. Could there be use-cases for other configurations?

Yes, I am fairly confident that these are static parameters. We have code to create both "Interface" and "Gateway" VPC endpoints ([L188-L200 of main.tf](https://github.com/cloudera-labs/terraform-cdp-modules/blob/42308bd0dd3d51e1100caf6a2457bb261b45703b/modules/terraform-cdp-aws-pre-reqs/main.tf#L188-L200) has the code for Gateway endpoints.

>These parameters are relevant only when create_vpc = true, is that correct?

Currently yes. We hit some issues when creating the endpoints for pre-existing VPCs (fixed in PR #34).
As we split this into a separate module (e.g. terraform-aws-vpc-endpoints) I think we might be able to allow the option of creating vpc endpoints when create_vpc = false.